### PR TITLE
loc handback

### PR DESCRIPTION
**Pull request by ericsciple**

### CODE REVIEW

File: Tasks/AzureWebPowerShellDeployment/Strings/resources.resjson/zh-TW/resources.resjson [hunk 1]
```diff
@@ -23,5 +23,7 @@
   "loc.messages.CannotupdatedeploymentstatusuniquedeploymentIdcannotberetrieved": "無法更新部署狀態，無法擷取獨一的 deploymentId",
   "loc.messages.Updatingdeploymenthistoryfordeployment0": "正在更新部署 {0} 的部署記錄",
   "loc.messages.Nofileswerefoundtodeploywithsearchpattern0": "找不到任何檔案可用搜尋模式 {0} 部署",
-  "loc.messages.Foundmorethanonefiletodeploywithsearchpattern0Therecanbeonlyone": "使用搜尋模式 {0} 找到一個以上要部署的檔案。但僅允許一個。"
+  "loc.messages.Foundmorethanonefiletodeploywithsearchpattern0Therecanbeonlyone": "使用搜尋模式 {0} 找到一個以上要部署的檔案。但僅允許一個。",
+  "loc.messages.FailedtodeployWebsiteError0": "無法部署網站。錯誤: {0}",
+  "loc.messages.DeprecatedWarning": "'Azure App Service: Classic' task will be deprecated soon. 'Azure App Service Deploy' task will replace 'Azure App Service: Classic' task and the recommendation is to migrate your Build or Release process to use the 'Azure App Service Deploy' task. Refer https://go.microsoft.com/fwlink/?LinkID=613750 for more details."
```
Comment (chshrikh) [inline Tasks/AzureWebPowerShellDeployment/Strings/resources.resjson/zh-TW/resources.resjson]: loc.messages.DeprecatedWarning - still this need to get translate, do we expect one more loc handback ?

**Comment by ericsciple** [comment]
@chshrikh i'll defer to your team to merge (since this is m109). mainly i just wanted to see what strings are back for m109. it looks like we are still waiting on a few translations.

i'll check in next week again to see if more strings are back.

**Comment by chshrikh** [comment]
sure I will merge this

File: Tasks/AzureWebPowerShellDeployment/Strings/resources.resjson/zh-TW/resources.resjson (hunk 1 above)
Comment (ericsciple) [inline Tasks/AzureWebPowerShellDeployment/Strings/resources.resjson/zh-TW/resources.resjson]: yes. the fact that the english string is getting generated is a good sign. it should mean the ball is in localizers court.

i just ran through handoff steps. so i'll check back next week to see if more strings are back. today is the last handoff. handback will continue next week, and likely wont meet the bar after that once we enter escrow mode.

**Comment by ericsciple** [comment]
another thing to consider is whether you need to bump the patch version on these files. i dont know if it has been changed between u1 and now.

**Comment by chshrikh** [comment]
I am increasing the patch version for both the task,

**Pull request merged**